.github/workflows/check-go-task.yml

```
@@ -127,6 +136,9 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
 
+      - name: cd to module
+        run: cd "${{ matrix.module.path }}"
+
       - name: Run go mod tidy
         run: go mod tidy
```

Shouldn't this work too? :thinking: 

```suggestion
      - name: Run go mod tidy
        working-directory: ${{ matrix.module.path }}
        run: go mod tidy
```

---

Great idea! I don't know that I have ever used that syntax, but it's perfect for this application.